Makefile

```diff
@@ -53,7 +53,7 @@ $(sandbox_pid): |$(daml_build_log)
 start_daml_server: $(sandbox_pid)
```

Comment:
Good to know, I will separate changes like this in the future. Technically this could have been three PRs as the real name mapping and the invite screens are separate as well (although certainly more related than the `Makefile`.